### ISSUE TITLE
Invalid token signature

### ISSUE THREAD

**Issue by timolins** [blake-simpson/expo-jwt#1]
It seems like this library doesn't align with the JWT standard.

```js
const jwt = require("expo-jwt").default

jwt.encode('My data', 'secret')
// eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.Ik15IGRhdGEi.XTD07Z/Mzh5uItOylrh4a2qtZspMMUIa2p7YyaueT3Q=
```

I tried to validate the token on [jwt.io](https://jwt.io/), but it can't verify the signature.

**Comment by blake-simpson** [comment]
Hi, thank you for the issue! I hadn't checked the signing via the JWT website yet.

It's interesting because I sign the header and body as specified but as you reported, something is wrong. I am guessing something to do with how CryptoJS transforms the body into a byte array.

I will look into this and for the meantime the library can at least encode/decode it's "own" tokens, and "real" JWT tokens can be decoded too.

**Comment by cfanboy** [comment]
The token cannot validate on jwt.io because the jwt.io is using the URL safe Base64 to encoded the payload, but expo-jwt used crypto-js/enc-base64.js is not URL safe.

**Comment by blake-simpson** [comment]
Thanks for the hint @cfanboy! In 75683011a3a4470 I've written a helper that will make the Crypto.js encoded Base64 URL safe and based on my testing the generated JWT validates correctly with the tokens on https://jwt.to/

I've just released that as version 1.0.3 if you need the update.

**Comment by cloudorbush** [comment]
Receiving same error `Invalid token signature`.

jwt.io decodes it fine.
